lib/Swagger/src/Configuration.php

```diff
@@ -89,7 +89,7 @@ class Configuration
      *
      * @var string
      */
-    protected $host = 'https://localhost/user-preference';
+    protected $host = 'https://localhost/';
```

Comment:
Looks that this default value is overwritten by ```setHost``` but would be great if User Preferences Service owners can confirm that.